local/core/etc/config-settings/p21/testnet.json

```diff
@@ -0,0 +1,306 @@
+{
+  "updated_entry": [
+    {
+      "contract_max_size_bytes": 65536
+    },
+    {
+      "contract_compute_v0": {
+        "ledger_max_instructions": 100000000,
+        "tx_max_instructions": 100000000,
+        "fee_rate_per_instructions_increment": 25,
+        "tx_memory_limit": 41943040
+      }
+    },
+    {
+      "contract_ledger_cost_v0": {
+        "ledger_max_read_ledger_entries": 40,
+        "ledger_max_read_bytes": 133120,
+        "ledger_max_write_ledger_entries": 25,
+        "ledger_max_write_bytes": 66560,
+        "tx_max_read_ledger_entries": 40,
+        "tx_max_read_bytes": 133120,
```

Comment:
shouldn't this be 200kB?

https://github.com/stellar/stellar-core/blob/master/soroban-settings/pubnet_phase3.json#L10

Reply:
Good catch. Looks like quite a few of these are out of date. I'll update to match https://github.com/stellar/stellar-core/blob/master/soroban-settings/testnet_settings_upgrade.json.

Reply:
Fixed. Thanks for catching this.